wordle-web/pages/leaderboard.vue

```diff
@@ -0,0 +1,63 @@
+<template>
+  <v-container fluid fill-height justify-center>
```

Comment:
Overall your leader board looks very nice.  It loads automatically, which is nice as well.